src/Main.cc

```diff
@@ -213,8 +213,14 @@ int main(int argc, char* argv[])
 {
   // This avoids problems with locale setting dependent
   // C functions like strtod (e.g. 0.5 vs 0,5).
-  setenv("LC_ALL", "en_US.UTF-8", /* override */ 1);
-  setenv("LANG", "en_US.UTF-8", /* override */ 1);
+  const char* locales[] = {"en_US.UTF-8", "C.UTF-8", "C"};
```

Comment:
It is a matter of taste, you decide, but you could avoid raw pointer and array:

```suggestion
  std::array locales{"en_US.UTF-8", "C.UTF-8", "C"};
```